setup.py

```diff
@@ -57,6 +57,7 @@
     "pytest-assume",  # sql counter check
     "decorator",  # sql counter check
     "protoc-wheel-0==21.1",  # Protocol buffer compiler, for Snowpark IR
+    "mypy-protobuf",  # used in generating typed Python code from protobuf for Snowpark IR
```

Comment:
I assume this has no licensing or security concerns, either. Should this require a specific minimum version?

Reply:
Yes, this is development requirements so should be fine.